pkgs/development/python-modules/pytest-factoryboy/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, pythonPackages }:
+
+buildPythonPackage rec {
+  pname = "pytest-factoryboy";
+  version = "2.0.3";
+
+  src = fetchFromGitHub {
+    owner = "pytest-dev";
+    repo = "pytest-factoryboy";
+    rev = version;
+    sha256 = "0m1snyybq2k51khlydhisq300vzys897vdbsicph628iran950hn";
+  };
+
+  # The project uses tox, which we can't. So we simply run pytest manually.
+  checkInputs = with pythonPackages; [
+    pytest
+    pytestcache
+    pytestcov
+    pytestpep8
+    mock
+  ];
+  checkPhase = ''
+    pytest -vv -l --pep8 --ignore docs
+  '';
+
+  propagatedBuildInputs = with pythonPackages; [ pytest inflection factory_boy ];
```

Move this line above the ``checkInputs``. Alphabetize
```suggestion
  propagatedBuildInputs = [ factory_boy inflection pytest ];
```

---

Done.